README.md

```diff
@@ -101,6 +101,100 @@ public class Main {
 }
 ```
 
+### Deploy Account example
+
+```java
+import org.junit.jupiter.api.Test;
+import com.swmansion.starknet.account.StandardAccount;
+import com.swmansion.starknet.crypto.StarknetCurve;
+import com.swmansion.starknet.data.ContractAddressCalculator;
+import com.swmansion.starknet.data.types.*;
+import com.swmansion.starknet.data.types.transactions.*;
+import com.swmansion.starknet.data.types.transactions.TransactionReceipt;
+import com.swmansion.starknet.provider.rpc.JsonRpcProvider;
+
+import java.math.BigInteger;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        // Create a provider for interacting with StarkNet
+        JsonRpcProvider provider = new JsonRpcProvider("https://example-node-url.com/rpc", StarknetChainId.TESTNET);
+
+        // Create an account interface
+        Felt privateKey = Felt.fromHex("0x12345");
+        Felt publicKey = StarknetCurve.getPublicKey(privateKey);
+
+        // Use the class hash of the desired account contract (i.e. the class hash of OpenZeppelin account contract)
+        Felt classHash = Felt.fromHex("0x058d97f7d76e78f44905cc30cb65b91ea49a4b908a76703c54197bca90f81773");
+        Felt salt = new Felt(789);
+        List<Felt> calldata = List.of(publicKey);
+        Felt address = ContractAddressCalculator.calculateAddressFromHash(
+                classHash,
+                calldata,
+                salt
+        );
+
+        StandardAccount account = new StandardAccount(address, privateKey, provider, Felt.ZERO);
+        
+        // Multiply the estimated fee by 10 to ensure the transaction will not fail due to insufficient funds
+        Felt maxFee = Felt.fromHex("0x11fcc58c7f7000");  // should be 10*fee from estimate deploy account fee
```

Comment:
Not sure whether we want a shorter example in README.md or an example that just works (the `starknet-jvm.md` version).